js/start.md

```diff
@@ -317,6 +317,10 @@ Create new AWS backend resources and pull the AWS services configuration into th
 $ amplify init
 ```
 
+<div id="react" class="tab-content">
+> When asked for the distribution directory (the directory that will be uploaded to S3), answer `dist`. If you miss this step you can run `amplify configure project` to change your dist directory setting.
```

Comment:
react uses 'build' as the distribution folder.

Reply:
Yes it does, but the getting started setup doesn't call out that it needs to be in the dist folder. If you go with the default build folder the quick start doesn't work and you end up with a 404. Can you call out what you'd like me to change here?

Reply:
@ryanjones it shouldn't need to be in the dist folder unless you are using some custom folder structure outside create-react-app? CRA puts the dist build in a `build` folder, which the CLI will use when publishing....is it possible that your build didn't get uploaded for some reason, potentially a cli bug or similar? Was your S3 bucket not populated?